src/framework/ui/view/uitheme.cpp

```diff
@@ -125,7 +125,7 @@ void UiTheme::initThemeValues()
     m_focusColor = themeValues[FOCUS_COLOR].toString();
 
     m_borderWidth = themeValues[BORDER_WIDTH].toReal();
-    //m_navCtrlBorderWidth = 2;
+    m_navCtrlBorderWidth = 2.0;
```

Comment:
Better to get this value from the current theme to avoid the magic number

Reply:
Yes that
I'm going to make it into a theme variable.
That will allow us to have different navigation border widths for different themes, which might come in handy

Reply:
Done :)